financialaid/api_test.py

```diff
@@ -41,7 +41,9 @@ class FinancialAidBaseTestCase(TestCase):
     @classmethod
     def setUpTestData(cls):
         with mute_signals(post_save):
-            cls.profile = ProfileFactory.create()
+            cls.profile = ProfileFactory.create(
+                country="US"
```

Comment:
This isn't strictly needed, but I wanted to ensure that this profile's auto-approval income threshold was the highest possible (75k).  Currently, this will always be the case since the fuzzy generator for profiles creates the country code with a suffix of "land", which will never match the two character county code.  I'm anticipating a future refactor where two letter country codes will be better enforced, and I'd rather not leave the potential for the generator to match an existing auto-approval country and fail tests.

Now given that, these tests likely need a refactor anyway so I'm happy to leave out this change in favor of doing it in a later refactor.  Thoughts?